superset/utils/core.py

```diff
@@ -817,7 +817,7 @@ def zlib_decompress_to_string(blob):
             decompressed = zlib.decompress(blob)
         else:
             decompressed = zlib.decompress(bytes(blob, "utf-8"))
-        return decompressed.decode("utf-8")
+        return decompressed.decode("utf-8") if decode else decompressed
```

Comment:
Oh wow [unrelated to this PR but] the `if PY3K` we should remove now since we force py3.6+